### PR TITLE
[elm/cs] Add Czech translation of Elm

### PR DESCRIPTION
**Pull request by robinpokorny**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Seriously, look at it now. Watch for quotes and double-check field names.

### CODE REVIEW

File: cs-cz/elm.html.markdown [hunk 1]
```diff
@@ -0,0 +1,372 @@
+---
+language: Elm
+contributors:
+    - ["Max Goldstein", "http://maxgoldste.in/"]
+translators:
+    - ["Robin Pokorný", "http://robinpokorny.com/"]
+lang: cs-cz
```
Comment (vendethiel) [inline cs-cz/elm.html.markdown]: please add a filename field (with the `-cz` suffix)

Comment (robinpokorny) [inline cs-cz/elm.html.markdown]: I updated it.

**Comment by vendethiel** [comment]
Thanks!

**Comment by robinpokorny** [comment]
@vendethiel, this translation is not yet live. Is there something I need to do?

**Pull request merged**